### DISCUSSION TITLE
Stackable Items in Boxes?

### DISCUSSION THREAD

**Discussion by beaudierman** [dvdoug/BoxPacker#554] [Q&A]
Does anyone have any suggestions on how to account for items that, if you have multiples of them, can stack inside of each other?   An example item would be cups, lids or plastic tubs.  These items that are stacked inside of one another take up the same width and depth, but add a small amount of height with each additional item.

**Comment by dvdoug** [comment]
In my (limited) experience, these tend to come pre-packaged e.g. 1 x 50 cups rather than 50 x 1 cup. If you're packing ad-hoc quantities, then I can't think of any clever solution other than to pre-group them yourself before passing to `Packer` to simulate this although of course then that gets into how many cups should be stacked together.....

That's a tricky one
